### PR TITLE
Add namespaces - For discussion ONLY

### PR DESCRIPTION
**Pull request by chadicus**
All classes in Stripe namespace
Use psr-4 autoloading in composer

### CODE REVIEW

File: lib/Account.php [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <?php
+namespace Stripe;
```
Comment (GrahamCampbell) [inline lib/Account.php]: I prefer the following formatting:

```php
<?php

namespace Stripe;

class ...
```

That style is more popular, for example, symfony use it.

Comment (chadicus) [inline lib/Account.php]: fixed.

**Comment by bkrausz** [comment]
:+1: :sheep: :smile:

**Comment by bkrausz** [comment]
Let me know if you wanted to do more work on this, since it's for the experimental branch and seems like an obvious next step I'm happy to have it in :).

**Comment by chadicus** [comment]
Whoa, didn't expect this to get merged. and indeed I have another PR.

**Pull request merged**